### PR TITLE
Created autoloaded Helpers.php

### PR DESCRIPTION
**Pull request by emir**
I grabbed some functions from Laravel's Helper for dotenv support

### CODE REVIEW

**Comment by s3b4stian** [comment]
Hi!
Great thing for semplify configuration. 
I think should be added a new namespace for helpers 'Linna\Support' and functions should be encapsulated inside a class.

**Comment by emir** [comment]
Why? I've already added them to composer's autoload and we don't need to create class for them. These are just helper functions. You can see the usage at linna/app.

**Comment by s3b4stian** [comment]
Uniformity with other parts of the Framework.

Functions could be very useful, but:
- Did you write the tests for this functions?
- Did you try to add the "type declaration" or the "return type"?
- Did you try to optimize and semplify the code?

File: src/Linna/Support/Helpers.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+ *
+ * @param  string $haystack
+ * @param  string|array $needles
+ * @return bool
+ * @link https://github.com/laravel/framework/blob/43eeceda41cfec19fca6765e4e74f01caec20c5c/src/Illuminate/Support/Str.php#L466
+ */
+function startsWith($haystack, $needles)
+{
+    foreach ((array)$needles as $needle) {
+        if ($needle !== '' && substr($haystack, 0, strlen($needle)) === (string)$needle) {
```
Comment (s3b4stian) [inline src/Linna/Support/Helpers.php:14]: Condition complexity could be reduced using: http://php.net/manual/en/function.strpos.php

**Comment by s3b4stian** [comment]
Hello!
I have implemented `Linna\Helper\Eng::get()`, please try it!

**Pull request closed without merging**